examples/flat-serialisation/anonymous-owner.json

```diff
@@ -0,0 +1,50 @@
+{
+  "version": "0.1",
+  "statementGroups": [
+    {
+      "statementID": "32e09b90-203d-42d1-bd77-a6465bca2552",
+      "beneficialOwnershipStatements": [
+        {
+          "statementID": "0e5c86db-f2b0-4e11-bff0-db9eee3db221",
+          "statementDate": "2017-07-18",
+          "subject": {
+            "subjectEntity": {
+              "type": "entityStatement",
+              "statementID": "77ee64ba-858e-4413-bfaf-991be2200f71"
+            }
+          },
+          "interestedParty": {
+            "interestedPerson": {
+              "type": "personStatement",
+              "statementID": "3154a26d-d29e-4db0-8815-866f57c75f0c"
+            }
+          }
+        }
+      ],
+      "entityStatements": [
+        {
+          "statementID": "77ee64ba-858e-4413-bfaf-991be2200f71",
+          "statementDate": "2017-07-18",
+          "name": "FERRERO UK LIMITED",
+          "type": "registeredEntity",
+          "jurisdiction": "GB",
+          "foundingDate": "1966-04-04",
+          "identifiers": [
+            {
+              "scheme": "GB-COH",
+              "id": "00876127"
+            }
+          ]
+        }
+      ],
+      "personStatements": [
+        {
+          "statementID": "3154a26d-d29e-4db0-8815-866f57c75f0c",
+          "statementDate": "2017-07-18",
+          "missingPersonType": "anonymousPerson",
```

Comment:
'Missing person' feels like it could be read the wrong way. 

Perhaps 'unspecifiedPerson'  or just 'personType':['anonymous','identified'] and something like 'redactionNote' if it would be legitimate to refer to this as redaction?

Reply:
Yes, I think there are two separate things that can be separated out here, in two different fields: the type of missing data we are dealing with (redacted, unknown, known)  and the reason the data is missing (company didn't file, company doesn't need to file, BO didn't respond, intermediary didn't respond, etc). I don't have that in schema form, yet, but it's next on my list.